spec/index.bs

```diff
@@ -4272,107 +4289,112 @@ available or applicable.
 Debug groups in a {{GPUCommandEncoder}} or {{GPUProgrammablePassEncoder}}
 must be well nested.
 
-### <dfn method for=GPUCommandEncoder>pushDebugGroup(groupLabel)</dfn> ### {#GPUCommandEncoder-pushDebugGroup}
-
-<div algorithm="GPUCommandEncoder.pushDebugGroup">
-    <strong>|this|:</strong> of type {{GPUCommandEncoder}}.
-
-    **Arguments:**
-      - {{USVString}} |groupLabel|
-
-    **Returns:** void
-
-    Marks the beginning of a labeled group of commands for the {{GPUCommandEncoder}}.
-
-    |groupLabel| defines the label for the command group.
-
-    On the [=Device timeline=], the following steps occur:
-
-        - If the [$GPUCommandEncoder.pushDebugGroup/Valid Usage$] rules are met:
-
-            - push |groupLabel| onto then end of |this|.{{GPUCommandEncoder/[[debug_group_stack]]}}.
-
-    <div class=validusage dfn-for=GPUCommandEncoder.pushDebugGroup>
-        <dfn abstract-op>Valid Usage</dfn>
-
-        - |this|.{{GPUCommandEncoder/[[state]]}} must be {{encoder state/open}}.
-    </div>
-
-</div>
-
-### <dfn method for=GPUCommandEncoder>popDebugGroup()</dfn> ### {#GPUCommandEncoder-popDebugGroup}
-
-<div algorithm="GPUCommandEncoder.popDebugGroup">
-    <strong>|this|:</strong> of type {{GPUCommandEncoder}}.
-
-    **Returns:** void
+<dl dfn-type="method" dfn-for="GPUCommandEncoder">
+    : <dfn>pushDebugGroup(groupLabel)</dfn>
+    ::
 
-    Marks the end of a labeled group of commands for the {{GPUCommandEncoder}}.
+        Marks the beginning of a labeled group of commands for the {{GPUCommandEncoder}}.
 
-    On the [=Device timeline=], the following steps occur:
+        <div algorithm=GPUCommandEncoder.pushDebugGroup>
+            **Called on:** {{GPUCommandEncoder}} |this|.
 
-        - If the [$GPUCommandEncoder.popDebugGroup/Valid Usage$] rules are met:
+            **Arguments:**
+            <pre class=argumentdef for="GPUCommandEncoder/pushDebugGroup(groupLabel)">
+                |groupLabel|: The label for the command group.
+            </pre>
 
-            - pop an entry off the end of |this|.{{GPUCommandEncoder/[[debug_group_stack]]}}.
+            **Returns:** void
 
-    <div class=validusage dfn-for=GPUCommandEncoder.popDebugGroup>
-        <dfn abstract-op>Valid Usage</dfn>
+            - Issue the following steps on the [=Device timeline=] of |this|:
+                <div class=device-timeline>
+                    <div class=validusage>
+                        - If |this|.{{GPUCommandEncoder/[[state]]}} is not {{encoder state/open}},
+                            generate a validation error and stop.
+                    </div>
+                    - Push |groupLabel| onto then end of |this|.{{GPUCommandEncoder/[[debug_group_stack]]}}.
+                </div>
```

Comment:
Sorry, I realize now I was unclear (and should have noticed on your screenshot of the styling issue). I had in mind was more like:

```suggestion
            Issue the following steps on the [=Device timeline=] of |this|:
            <div class=device-timeline>
                - If any of the following conditions are unsatisfied, generate a validation error and stop.
                    <div class=validusage>
                        - |this|.{{GPUCommandEncoder/[[state]]}} is {{encoder state/open}}.
                    </div>
                - Push |groupLabel| onto then end of |this|.{{GPUCommandEncoder/[[debug_group_stack]]}}.
            </div>
```

In particular, inlining some-steps-in-a-box next to some-other-steps-outside-a-box breaks the list flow and makes it less clear it's all part of the same list of steps. WDYT?

Maybe this obviates the need for the css fix, but it doesn't hurt anything so may as well keep it?

Reply:
Ah! I see what you meant now, and why I misread it. Updated the PR to account for the _actual_ suggestion. 😁